lmctl/project/handlers/brent/brent_src.py

```diff
@@ -505,8 +527,36 @@ def __create_lifecycle(self, journal, source_request, file_ops, source_tree, lif
                 ptype='string')
             descriptor.add_property('localizationLanguage', description='Localization language of the NS to be instantiated', ptype='string')
             descriptor.insert_lifecycle('Create')
-            descriptor.insert_lifecycle('Install')
-            descriptor.insert_lifecycle('Uninstall')
+            descriptor.insert_lifecycle('Update')
```

Comment:
These update the lifecycles included when generating a sol005 resource - was it intentional to change these?

Reply:
It happened when I was merging the code line by line. Just committed the fixed code.